static/library-scripts/docker-in-docker-debian.sh

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env bash
```

Comment:
Just double-checking... This file is taken as-is from the MS repo, correct? No modifications?

Reply:
confirmed, j'ai rien changé